src/SFA.DAS.LevyTransferMatching.Web/Models/Pledges/GetApplicationViewModel.cs

```diff
@@ -1,12 +1,30 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Globalization;
+using System.Linq;
 using SFA.DAS.LevyTransferMatching.Infrastructure.Dto;
+using SFA.DAS.LevyTransferMatching.Infrastructure.ReferenceData;
+using SFA.DAS.LevyTransferMatching.Web.Extensions;
+using StructureMap.Query;
 
 namespace SFA.DAS.LevyTransferMatching.Web.Models.Pledges
 {
-    public class ApplicationViewModel
+    public class GetApplicationViewModel
```

Comment:
`ApplicationViewModel` looks correct to me; is there a reason for the name change

Reply:
Confusion, I thought the model as well as the endpoint needed changing.